luma/core/framebuffer.py

```diff
@@ -6,15 +6,17 @@
 Different implementation strategies for framebuffering
```

Comment:
copyright date in header needs to be updated.

Reply:
done